documentation/book/con-tls-within-the-cluster.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='tls-within-the-cluster-{context}']
+= TLS within the cluster
```

Comment:
Why is the title _TLS within the cluster_? It refers to all the TLS aspects inclusing clients and external clients. From the title I have assumed that this will talk only about replication and Zookeeper communication. Maybe we should rename it.

Reply:
To what though? "TLS connections"?

Reply:
Yeah, that sounds better.